core/state_processor.go

```diff
@@ -774,21 +747,20 @@ func (p *ParallelStateProcessor) execInParallelSlot(slotIndex int, txReq *Parall
 		p.debugConflictRedoNum++
 		// re-run should not have conflict, since it has the latest world state.
 		redoResult := &ParallelTxResult{
-			redo:         true,
 			updateSlotDB: true,
 			keepSystem:   systemAddrConflict,
 			slotIndex:    slotIndex,
 			txReq:        txReq,
 			receipt:      receipt,
 			err:          err,
 		}
-		p.paraTxResultChan <- redoResult
+		p.txResultChan <- redoResult
 		slotDB = <-p.slotState[slotIndex].slotdbChan
 		slotDB.Prepare(tx.Hash(), block.Hash(), txIndex)
 		gpSlot = new(GasPool).AddGas(slotGasLimit)
 		evm, result, err = applyTransactionStageExecution(msg, gpSlot, slotDB, vmenv)
 		if err != nil {
-			panic(fmt.Sprintf("Stage Execution conflict redo, error %v", err))
+			log.Error("Stage Execution conflict redo, error", err)
```

Comment:
If this happens, I think we  should stop the process of this block?

Reply:
Yes, just let it go here; The block will be marked as forked later;